### PR TITLE
Update docs for new config classes

### PR DESCRIPTION
**Pull request by collindutter**
----
📚 Documentation preview 📚: https://griptape--218.org.readthedocs.build//218/

### CODE REVIEW

**Comment by collindutter** [comment]
Should definitely merge in the other PRs first.

File: docs/griptape-framework/structures/config.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+The [StructureConfig](../../reference/griptape/config/structure_config.md) class allows for the customization of Structures within Griptape, enabling specific settings such as Drivers to be defined for Tasks. 
+
+### Premade Configs
+
+Griptape provides predefined [StructureConfig](../../reference/griptape/config/structure_config.md)'s for widely used services that provide APIs for most Driver types Griptape offers.
+
+#### OpenAI
+
+The [OpenAI Structure Config](../../reference/griptape/config/structure_config.md#griptape.config.structure_config.OpenStructureAIConfig) provides default Drivers for OpenAI's API's. This is the default config for all Structures.
```
Comment (andrewfrench) [inline docs/griptape-framework/structures/config.md:11]: API's -> APIs

File: docs/griptape-framework/structures/config.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+agent = Agent(
+    config=OpenAIStructureConfig()
+)
+
+agent = Agent() # This is equivalent to the above
+```
+
+#### Amazon Bedrock
+The [Amazon Bedrock Structure Config](../../reference/griptape/config/structure_config.md#griptape.config.structure_config.AmazonBedrockStructureConfig) provides default Drivers for Amazon Bedrock's API's.
```
Comment (andrewfrench) [inline docs/griptape-framework/structures/config.md:25]: API's -> APIs

**Review by andrewfrench — COMMENTED** [review]
Nits

**Comment by andrewfrench** [comment]
The Marqo test error is annoying flakiness we'll have to contend with at some points, but the other failures seem legit to me.

**Review by andrewfrench — APPROVED** [review]

**Pull request merged**